.github/workflows/ci.yml

```
@@ -1,6 +1,6 @@
 name: Test
 
-on: [push]
+on: [push, workflow_call]
```

I didn't realise this isn't also running on pull_request. Can you add that in too?
```suggestion
on: [push, pull_request, workflow_call]
```